client/dashboard/sites/performance/page-selector.tsx

```diff
@@ -1,38 +1,32 @@
-import { ComboboxControl, __experimentalVStack as VStack } from '@wordpress/components';
+import { CustomSelectControl } from '@wordpress/components';
 import { useViewportMatch } from '@wordpress/compose';
 import { __ } from '@wordpress/i18n';
 import { useMemo } from 'react';
-import { Text } from '../../components/text';
 import type { SitePerformancePage } from '@automattic/api-core';
-
+import type { CSSProperties } from 'react';
 import './page-selector.scss';
 
 interface PageOption {
-	url: string;
-	path: string;
-	label: string;
-	value: string;
-	disabled: boolean;
-	wpcom_performance_report_hash: string;
+	key: string;
+	name: string;
+	hint?: string;
+	style?: CSSProperties;
 }
 
-/**
- * Map a PerformanceProfilerPage to a PageReport
- * @param page - The PerformanceProfilerPage to map
- * @param siteUrl - The URL of the site
- * @returns The PageReport
- */
+const defaultStyle = {
+	lineHeight: 1.2,
+};
+
 function mapPageToPageOption( page: SitePerformancePage, siteUrl: string ): PageOption {
-	let path = page.link.replace( siteUrl ?? '', '' );
-	path = path.length > 1 ? path.replace( /\/$/, '' ) : path;
+	const key = page.id.toString();
+	const name = page.title.rendered || __( 'No Title' );
```

Comment:
Let's use sentence case.

```suggestion
	const name = page.title.rendered || __( 'No title' );
```